File: csharp/examples/GetOrderItemsRequiringAttention.cs
using System;
namespace Examples.examples
{
    // Example of GET /order-items/requiring-attention
    
    public class GetOrderItemsRequiringAttention : BaseRequest
    {
        private string _companyId;

        public GetOrderItemsRequiringAttention()
		{
            dotenv.net.DotEnv.Load();
            _companyId = Environment.GetEnvironmentVariable("COMPANY_ID");
            Console.WriteLine($"GetOrderItemsRequiringAttention: _companyId={_companyId}");
        }

        public GetOrderItemsRequiringAttention(string companyId)
        {
            _companyId = companyId;
        }

        public override string SendRequest()
        {
            return GetRequest($"order-items/requiring-attention?company_ids={_companyId}");
        }
    }
}

File: php/examples/get_order_items_requiring_attention.php
<?php
require_once 'base_request.php';

// Example of GET /order-items/requiring-attention

$debug = true;
$company_id   = $_ENV['COMPANY_ID'];
$request_data = '';
$request_params = ['company_ids' => [$company_id]];

send_request('GET', '/order-items/requiring-attention', $request_params, $request_data);